### PR TITLE
Jobergum/streaming example

### PR DESCRIPTION
**Pull request by jobergum**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: docs/sphinx/source/examples/scaling-personal-ai-assistants-with-streaming-mode-cloud.ipynb [hunk 1]
```diff
@@ -0,0 +1,1672 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f77beb5a",
+   "metadata": {},
+   "source": [
+    "For the above query request, Vespa searched the `default` fieldset which we defined in the schema to match against several fields including the body and the subject. The `default`\n",
+    "rank-profile calculated the relevance score as the sum of three rank-features: `nativeRank(body)` + `nativeRank(subject)` + `freshness(timestamp)` and the result of this computation is the\n",
+    "`relevance` score of the hit. In addition, we also asked for Vespa to return `match-features` that can be used to debug the `relevance` score or for feaature logging. \n",
```
Comment (kkraune) [inline docs/sphinx/source/examples/scaling-personal-ai-assistants-with-streaming-mode-cloud.ipynb:1155]:
```suggestion
    "`relevance` score of the hit. In addition, we also asked for Vespa to return `match-features` that can be used to debug the `relevance` score or for feature logging. \n",
```

File: docs/sphinx/source/examples/scaling-personal-ai-assistants-with-streaming-mode-cloud.ipynb [hunk 2]
```diff
@@ -0,0 +1,1672 @@
...
+   "metadata": {},
+   "source": [
+    "## LlamaIndex Retrievers Introduction\n",
+    "\n",
+    "Now, we have a basic Vespa app using streaming mode up. For building an end to end assistant, we likely want to\n",
+    "use a LLM framework like [LangChain](https://www.langchain.com/) or [LLamaIndex](https://www.llamaindex.ai/). In this example \n",
+    "we use LLamaIndex retrievers. \n",
+    "\n",
+    "LlamaIndex [retriever](https://gpt-index.readthedocs.io/en/latest/core_modules/query_modules/retriever/root.html)\n",
+    "abstraction allows developers to add custom retrivers that retrieve information in Retrieval Augmented Generation (RAG) pipelines. \n",
```
Comment (kkraune) [inline docs/sphinx/source/examples/scaling-personal-ai-assistants-with-streaming-mode-cloud.ipynb:1279]:
```suggestion
    "abstraction allows developers to add custom retrievers that retrieve information in Retrieval Augmented Generation (RAG) pipelines. \n",
```

File: docs/sphinx/source/examples/scaling-personal-ai-assistants-with-streaming-mode-cloud.ipynb [hunk 3]
```diff
@@ -0,0 +1,1672 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1188a1e3",
+   "metadata": {},
+   "source": [
+    "The above defines a `PersonalAssistantVespaRetriever` which accepts most importantly a [pyvespa](https://pyvespa.readthedocs.io/en/latest/)\n",
+    "`Vespa` application instance. \n",
+    "\n",
+    "The YQL specifies a hybrid retrieval query that retrieves both using embedding based retrieval (vector search) \n",
```
Comment (kkraune) [inline docs/sphinx/source/examples/scaling-personal-ai-assistants-with-streaming-mode-cloud.ipynb:1383]:
```suggestion
    "The YQL specifies a hybrid retrieval query that retrieves both using embedding-based retrieval (vector search) \n",
```

**Review by kkraune — DISMISSED** [review]
💯

**Comment by kkraune** [comment]
also add the (Vespa Cloud) logo on top like the other notebooks

File: docs/sphinx/source/examples/scaling-personal-ai-assistants-with-streaming-mode-cloud.ipynb [hunk 4]
```diff
@@ -0,0 +1,1672 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "id": "ed430761",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vespa_application_package.to_files(\"app-directory\")\n",
```
[the remark this answers is not in the document]
Comment (jobergum) [reply docs/sphinx/source/examples/scaling-personal-ai-assistants-with-streaming-mode-cloud.ipynb:299]: This is now done by pyvespa if the directory does not exist

**Review by kkraune — COMMENTED** [review]
👍

**Review by kkraune — APPROVED** [review]
💯

**Pull request merged**